docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -612,6 +612,21 @@ iceberg.snowflake-catalog.password=secret
 iceberg.snowflake-catalog.database=db
 ```
 
+When using the Snowflake catalog, data management tasks such as creating tables,
+must be performed in Snowflake because using the catalog from external systems
+like Trino only supports `SELECT` queries as read operations.
+
+Additionally, the [Snowflake-created Iceberg
+tables](https://docs.snowflake.com/en/sql-reference/sql/create-iceberg-table-snowflake)
+do not expose partitioning information, which prevents efficient parallel reads
+and therefore can have significant negative performance implications.
+
+The Snowflake catalog does not support [view management](sql-view-management) or
```

Comment:
nit: Snowflake catalog  -> Snowflake Iceberg catalog ?

Reply:
I dont think so .. we dont say REST Iceberg catalog, Nessie Iceberg catalog, or JDBC Iceberg catalog either ..  currently its all consistent and relies on the context that they are all Iceberg catalogs (metastores)

Reply:
Thought of it to add more clarification as to Snowflake catalog could have different meaning in Trino context.

Reply:
Thought of it to add more clarification as tSnowflake catalog could have different meaning in Trino context.